assets/src/edit-story/components/dropDown/index.js

```diff
@@ -170,18 +170,25 @@ function DropDown({ options, value, onChange, disabled, ariaLabel }) {
   const [isOpen, setIsOpen] = useState(false);
   const [searchValue, setSearchValue] = useState('');
   const [focusedValue, setFocusedValue] = useState(null);
+  const isNullOrUndefined = (item) => {
+    if (item === null || item === undefined) return true;
+    return false;
```

Comment:
This could be just `return item === null || item === undefined;`

Reply:
Correct. Thank you